sdk/core/test/unit/services/WalletStorageService.test.ts

```diff
@@ -99,277 +97,159 @@ describe('WalletStorageService', () => {
-      expect(response.length).to.eql(250)
-      expect(response).to.eql(Array(250).fill(signedCredential))
-    })
-
-    it('should work for whole number of pages', async () => {
-      await authorizeVault()
-
-      nock(STAGING_AFFINIDI_VAULT_URL)
-        .get(fetchCredentialsBase + '0/99')
-        .reply(200, Array(100).fill(signedCredential))
-
-      nock(STAGING_AFFINIDI_VAULT_URL)
-        .get(fetchCredentialsBase + '100/199')
-        .reply(200, Array(100).fill(signedCredential))
-
-      nock(STAGING_AFFINIDI_VAULT_URL)
-        .get(fetchCredentialsBase + '200/299')
-        .reply(200, [])
-
-      const walletStorageService = createWalletStorageService()
-      const response = await walletStorageService.fetchAllBlobs()
-
-      expect(response).to.be.an('array')
-      expect(response.length).to.eql(200)
-      expect(response).to.eql(Array(200).fill(signedCredential))
-    })
-
-    it('should handle deleted values when fetching all', async () => {
-      await authorizeVault()
-
-      {
-        const expectedResult = Array(100).fill(signedCredential)
-        expectedResult[2] = { cyphertext: null }
-        const expectedPath = `${fetchCredentialsBase}0/99`
-        nock(STAGING_AFFINIDI_VAULT_URL).get(expectedPath).reply(200, expectedResult)
-      }
-
-      {
-        const expectedResult = Array(100).fill(signedCredential)
-        expectedResult[90] = { cyphertext: null }
-        const expectedPath = `${fetchCredentialsBase}100/199`
-        nock(STAGING_AFFINIDI_VAULT_URL).get(expectedPath).reply(200, expectedResult)
-      }
+    const walletStorageService = createWalletStorageService()
+    await walletStorageService.deleteCredentialById(signedCredential.id)
+  })
 
-      {
-        const expectedResult = Array(50).fill(signedCredential)
-        expectedResult[30] = { cyphertext: null }
-        const expectedPath = `${fetchCredentialsBase}200/299`
-        nock(STAGING_AFFINIDI_VAULT_URL).get(expectedPath).reply(200, expectedResult)
-      }
+  it('#getCredentialsById not found in Affinidi vault and checking in Bloom vault', async () => {
+    sinon
+      .stub(AffinidiVaultStorageService.prototype, 'deleteCredentialById')
+      .withArgs(signedCredential.id, region)
+      .throws({ code: 'AVT-2' })
 
-      {
-        const expectedPath = `${fetchCredentialsBase}300/399`
-        nock(STAGING_AFFINIDI_VAULT_URL).get(expectedPath).reply(200, [])
-      }
+    sinon
+      .stub(BloomVaultStorageService.prototype, 'deleteCredentialById')
+      .withArgs(signedCredential.id, region)
+      .resolves()
 
-      const walletStorageService = createWalletStorageService()
-      const allBlobs = await walletStorageService.fetchAllBlobs()
-
-      expect(allBlobs.length).to.eql(247)
-      expect(allBlobs).to.eql(Array(247).fill(signedCredential))
-    })
+    const walletStorageService = createWalletStorageService()
+    await walletStorageService.deleteCredentialById(signedCredential.id)
```

Comment:
should probably also check here that the second stub is called at least

Reply:
yes, good point. Ideally we should check all the stubs are getting called, but should check for cases when nothing to check for sure.